recipe/build.sh

```diff
@@ -1,3 +1,6 @@
+#!/bin/bash
```

Comment:
Could you remove these two lines unless this is to some style?

Reply:
It's more explicit about the intended usage, but I don't mind removing the `#!/bin/bash`. I do not want to remove `set -ex` because that says to log the output and fail on error (rather than spuriously keep going)

Reply:
fine with me